scheduler/cloudformation/cloudformation.go

```diff
@@ -235,34 +246,70 @@ func (s *Scheduler) submit(ctx context.Context, tx *sql.Tx, app *scheduler.App,
 		return &templateValidationError{templateURL: url, err: err, templateBody: buf}
 	}
 
+	var stackParameters []*cloudformation.Parameter
+	var create bool
+	resp, err := s.cloudformation.DescribeStacks(&cloudformation.DescribeStacksInput{
+		StackName: aws.String(stackName),
+	})
+	if err, ok := err.(awserr.Error); ok && err.Message() == fmt.Sprintf("Stack with id %s does not exist", stackName) {
+		create = true
+	} else if err == nil {
+		create = false
+		stackParameters = resp.Stacks[0].Parameters
```

Comment:
ah cool. i was initially going to go with UseExistingTemplate, but was worried about having to ensure we submit all the parameters.